tests/dtls-client.c

```diff
@@ -482,31 +484,45 @@ main(int argc, char **argv) {
     case 'v' :
       log_level = strtol(optarg, NULL, 10);
       break;
+    case -1 :
+      /* handle arguments */
+      if (!dst.size) {
+        /* resolve destination address where server should be sent */
+        res = resolve_address(argv[optind++], &dst.addr.sa);
+        if (res < 0) {
+          dtls_emerg("failed to resolve address\n");
+          exit(-1);
+        }
+        dst.size = res;
+      } else if (!dst_port) {
+        dst_port = atoi(argv[optind++]);
```

Comment:
For code readability, I think this should say `/* Optional second argument */`

Reply:
Done.